packages/web-components/src/components/va-textarea/va-textarea.tsx

```diff
@@ -233,14 +248,25 @@ export class VaTextarea {
           <div class="input-wrap">
             {label && (
               <label htmlFor="input-type-textarea" id="input-label" class={labelClass} part="label">
-                {label}
-                {required && (
-                  <span class="usa-label--required">
-                    {' '}
-                    {i18next.t('required')}
+               {HeaderLevel ? (
+                  <HeaderLevel part="header" aria-describedby={headerAriaDescribedbyId}>{label}</HeaderLevel>
+                ) : (
+                  label
+                )}&nbsp;
+                {
+                  useFormsPattern === 'multiple' && (
+                    <span id="header-message" class="sr-only">
+                      {label}
+                    </span>
+                  )
+                }
+                {headerAriaDescribedby && (
+                  <span id="header-message" class="sr-only">
+                    {headerAriaDescribedby}
                   </span>
                 )}
-                {hint && <span class="usa-hint">{hint}</span>}
+                {required && <span class="usa-label--required" part="required">{i18next.t('required')}</span>}
```

Comment:
Was this `part="required"` added intentionally?

Reply:
No it was copied. I'll remove it.